src/solvers/smt2_incremental/smt_to_smt2_string.cpp

```diff
@@ -13,7 +13,6 @@
 
 #include <functional>
 #include <iostream>
-#include <sstream>
```

Comment:
`std::stringstream` is used in the `smt_to_smt2_string` function (all overloads) and is defined in the `<sstream>`  header. Therefore this include should not be removed, as it is used. I assume this file still compiles without the include due to transitive includes of some type. My understanding is that some of these transitive includes can be in implementations of the STL but not required by the standard.

Reply:
I have now added `// IWYU pragma: keep` instead.